code/modules/surgery/organs/tongue.dm

```diff
@@ -318,3 +318,53 @@
 /obj/item/organ/tongue/ethereal/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_ethereal
+
+//Sign Language Tongue - yep, that's how you speak sign language.
+/obj/item/organ/tongue/tied
+	name = "tied tongue"
+	desc = "If only one had a sword so we may finally untie this knot."
+	say_mod = "signs"
+	icon_state = "tonguetied"
+	modifies_speech = TRUE
+	organ_flags = ORGAN_UNREMOVABLE
+
+/obj/item/organ/tongue/tied/Insert(mob/living/carbon/M)
+	. = ..()
+	M.verb_ask = "signs"
+	M.verb_exclaim = "signs"
+	M.verb_whisper = "subtly signs"
+	M.verb_sing = "rythmically signs"
+	M.verb_yell = "emphatically signs"
+	ADD_TRAIT(M, TRAIT_SIGN_LANG, "tongue")
+	REMOVE_TRAIT(M, TRAIT_MUTE, "tongue")
+
+/obj/item/organ/tongue/tied/Remove(mob/living/carbon/M, special = 0)
+	..()
+	M.verb_ask = initial(verb_ask)
+	M.verb_exclaim = initial(verb_exclaim)
+	M.verb_whisper = initial(verb_whisper)
+	M.verb_sing = initial(verb_sing)
+	M.verb_yell = initial(verb_yell)
+	REMOVE_TRAIT(M, TRAIT_SIGN_LANG, "tongue") //People who are Ahealed get "cured" of their sign language-having ways. If I knew how to make the tied tongue persist through aheals, I'd do that.
+
+//Thank you Jwapplephobia for helping me with the literal hellcode below
+
+/obj/item/organ/tongue/tied/handle_speech(datum/source, list/speech_args)
+	var/new_message
+	var/message = speech_args[SPEECH_MESSAGE]
+	var/mob/living/carbon/M = owner
+	new_message = message
+	if(findtext(message, "!"))
+		if(!findtext(message, "?"))
+			M.visible_message("<span class='notice'>[M] raises [M.p_their()] eyebrows.</span>")
+		else
+			M.visible_message("<span class='notice'>[M] lowers one of [M.p_their()] eyebrows, raising the other.</span>")
+		new_message = replacetext(new_message, "!", "")
+
+	if(findtext(message, "?"))
+		if(!findtext(message, "!"))
+			M.visible_message("<span class='notice'>[M] lowers [M.p_their()] eyebrows.</span>")
+		new_message = replacetext(new_message, "?", "")
```

Comment:
Why not do something like
```
var/exclamation_found = findtext(message, "!")
var/question_found = findtext(message, "?")
```
then use use the results of the find text operation for the checks, rather then rerunning them every time

Reply:
Did that make it better? I hope the `else if`s aren't too bad, if I didn't put them in it would put both the eyebrow raising and lowering at the same time, rather than only one.

Reply:
Don't worry about the else if man, it's a meme, you're doin fine